widget/storybook/src/components/Icons/index.ts

```diff
@@ -1,25 +1,36 @@
 import {
   AddIcon,
+  AffiliateIcon,
   AutorenewIcon,
   AutoThemeIcon,
   BorderRadiusIcon,
   BridgesIcon,
+  BugReportIcon,
   BullhornIcon,
+  CalendarIcon,
   ChainsIcon,
+  ChartIcon,
   ChevronDownIcon,
   ChevronLeftIcon,
   ChevronRightIcon,
   ChevronUpIcon,
   CloseIcon,
   ColorsIcon,
   CompleteIcon,
+  CookieIcon,
   CopyIcon,
   CosmosCategoryIcon,
+  CreditCardIcon,
   CustomColorsIcon,
+  CustomerSupportIcon,
+  CustomTokensZeroStateDarkIcon,
+  CustomTokensZeroStateIcon,
   DarkModeIcon,
   DeleteIcon,
   DesktopIcon,
+  DisconnectIcon,
   DiscordIcon,
+  DocumentationIcon,
   DocumentIcon,
   DoneIcon,
   ErrorIcon,
```

Comment:
You only added one icon, so why were all these other icon names included in your changes?

Reply:
I also added the remaining icons that were not included in the Storybook.